src/ServiceControl/ExternalIntegrations/Config/AdapterElement.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace ServiceControl.ExternalIntegrations.Config
+{
+    using System.Configuration;
+
+    public class AdapterElement : ConfigurationElement
+    {
+        [ConfigurationProperty("Name", IsRequired = true)]
+        public string Name
+        {
+            get { return (string)this["Name"]; }
+            set { this["Name"] = value; }
```

Comment:
Are we assuming that these adapters will only ever be NServiceBus addresses? Is it too early to consider other types of adapters?